### ISSUE TITLE
doc improvement 

### ISSUE THREAD

**Issue by sinsunsan** [vasyabigi/angular-slick#32]
It took me some time figure out that we need to provide angular-slick with a ng-repeat of an array of div.... 
For example it isn't possible not to use ng-repeat ie use only existing dom content inside slick container. 
Or at least it hasn't recognized my structure. 
There seem to have no way to telle slick to use container with the class .... 
There is a lack of documentation at the end of your description that make it hard how to start. 
Even with the demo it is not self understandable

**Comment by 1ambda** [comment]
+1 this [SO question](http://stackoverflow.com/questions/25087271/using-slick-caroussel-inside-a-template) helped me a lot

**Comment by nguyenDalex** [comment]
+1 more documentation!

**Comment by JamesIves** [comment]
I completely agree that the documentation for this is really poor. It took quite a bit of digging around SO to figure out how to get this to work properly. I'd gladly contribute some documentation changes if they would be welcomed by the author.
